Dockerfile-arm

```diff
@@ -1,6 +1,6 @@
 # first need to get the terrafrom binary and extract it
 FROM alpine:3.10 AS terraform_installer
-ADD https://releases.hashicorp.com/terraform/0.12.12/terraform_0.12.12_linux_arm.zip /tmp/
```

Comment:
Good catch but if it could be in it's own pull request (alongside the matching .drone.yml change) I'll be able to merge it in without having to wait until we figure out the more problematic concurrency issue (plus it will keep the rest of the pull request cleaner)